Examples/FluxClientPocoExample.cs

```diff
@@ -6,11 +6,11 @@ namespace Examples
 {
     public static class FluxClientPocoExample
     {
-        public static async Task Main(string[] args)
+        public static async Task Main()
         {
             var options = new FluxConnectionOptions("http://127.0.0.1:8086");
 
-            using var fluxClient = FluxClientFactory.Create(options);
+            using var fluxClient = new FluxClient(options);
```

Comment:
Please rename var to `client`.
```suggestion
            using var client = new FluxClient(options);
```